### PR TITLE
5 setup routing on icosium

### LINKED ISSUE #5: Setup Routing on icosium
# description
set up react-router on icosium app
https://reactrouter.com/en/main/routers/create-browser-router
## acceptance criteria
- router installed
- should use browserRouter
- Router component created
- Routing between 2 pages

### PR DESCRIPTION
**Pull request by osminosm**
closes : #5

### CODE REVIEW

**Comment by badido39** [comment]
tested localy , 5/5

File: package.json [hunk 1]
```diff
@@ -39,8 +40,8 @@
     "@types/express": "^4.17.14",
     "autoprefixer": "^10.4.12",
     "concurrently": "^7.5.0",
+    "nodemon": "^2.0.20",
     "postcss": "^8.4.18",
-    "tailwindcss": "^3.1.8",
-    "nodemon": "^2.0.20"
```
Comment (badido39) [inline package.json:44]: dont need nodemon anymore ?

**Review by badido39 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (osminosm) [reply package.json:44]: good catch, i dont know how i removed it

Comment (badido39) [reply package.json:44]: ;)

**Review by badido39 — APPROVED** [review]

**Review by nazimboudeffa — APPROVED** [review]

**Pull request merged**